### PR TITLE
Performance scripts fix.

### PR DESCRIPTION
**Pull request by navin22**
- Adding TestAssets project to the TestPlatform.sln. So that it's restored and built.
- Fixing some bugs in perf.ps1 script. And adding it to the sln file.
- Added myget feeds of the adapters to the Nuget.config file.

### CODE REVIEW

File: TestPlatform.sln [hunk 1]
```diff
@@ -167,7 +167,21 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "DataCollectors", "DataColle
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.TestPlatform.Extensions.EventLogCollector.UnitTests", "test\DataCollectors\Microsoft.TestPlatform.Extensions.EventLogCollector.UnitTests\Microsoft.TestPlatform.Extensions.EventLogCollector.UnitTests.csproj", "{21DB138B-85B7-479E-91FE-01E0F972EC56}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EventLogUnitTestProject", "test\TestAssets\EventLogUnitTestProject\EventLogUnitTestProject.csproj", "{826CD5AF-44FA-40F6-B731-3980CADED8C0}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EventLogUnitTestProject", "test\TestAssets\EventLogUnitTestProject\EventLogUnitTestProject.csproj", "{826CD5AF-44FA-40F6-B731-3980CADED8C0}"
```
Comment (codito) [inline TestPlatform.sln]: Can we add these into a separate solution please?

File: scripts/perf/perf.ps1 [hunk 2]
```diff
@@ -209,8 +209,8 @@ function Measure-DiscoveryTime($commandtorun, $payload)
     {
         Write-Log "Discovering Tests in $($payload.containerPath) using $($payload.currentRunner)"
         $result = Get-TimeTaken $commandtorun
-        $result.Action = "Discovery"
-        $result.Goal = $payload.discoverygoal[$payload.runners.IndexOf($payload.currentRunner)]
+        $result.Action = "Discovery"
```
Comment (codito) [inline scripts/perf/perf.ps1]: nit: please clean up the extra spaces

Comment (navin22) [reply scripts/perf/perf.ps1]: Done.

File: TestPlatform.sln (hunk 1 above)
Comment (navin22) [reply TestPlatform.sln]: I have added an Extra solution called `TestAssets.sln` Add most of the big projects to it. Please take a look.

File: test/TestAssets/XUnitAdapterPerfTestProject/XUnitAdapterPerfTestProject.csproj [hunk 3]
```diff
@@ -2,7 +2,7 @@
   <!-- Package dependency versions -->
   <Import Project="..\..\..\scripts\build\TestPlatform.Dependencies.props" />
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
```
Comment (smadala) [inline test/TestAssets/XUnitAdapterPerfTestProject/XUnitAdapterPerfTestProject.csproj]: Latest Xunit adapter supports `net451`?

**Review by smadala — APPROVED** [review]

**Review by codito — APPROVED** [review]

File: test/TestAssets/XUnitAdapterPerfTestProject/XUnitAdapterPerfTestProject.csproj (hunk 3 above)
Comment (navin22) [reply test/TestAssets/XUnitAdapterPerfTestProject/XUnitAdapterPerfTestProject.csproj]: Good catch Sathya, No they don't. I have changed it. Thanks.

**Comment by khilan2goel** [comment]
@dotnet-bot test Windows_NT / Debug Build

**Pull request merged**